packages/utils-compat-data/src/helpers.ts

```diff
@@ -1,5 +1,5 @@
 import { Identifier } from 'mdn-browser-compat-data/types';
-import { vendor } from 'postcss';
+import {getVendorPrefix, getUnprefixed} from '@hint/utils-css';
```

Comment:
Nit for consistency with other imports:
```suggestion
import { getVendorPrefix, getUnprefixed } from '@hint/utils-css';
```